cmd/spacelift-vcs-agent/main.go

```diff
@@ -267,7 +268,7 @@ var app = &cli.App{
 }
 
 func main() {
-	if err := app.Run(os.Args); err != nil {
+	if err := app.Run(context.Background(), os.Args); err != nil {
```

Comment:
Just to check my understanding of it, is the library now not bootstraping a context but instead we have to pass our own? That's a great change.

Reply:
yeah: https://cli.urfave.org/v3/getting-started/